Data/models/Bookmark.swift

```diff
@@ -459,7 +523,9 @@ extension Bookmark {
             if isFolder {
                 removeFolderAndSendSyncRecords(uuid: syncUUID)
             } else {
-                Sync.shared.sendSyncRecords(action: .delete, records: [self])
+                DataController.perform(context: context) { context in
+                    Sync.shared.sendSyncRecords(action: .delete, records: [self], context: context)
```

Comment:
A bit out of scope of this PR but this was throwing threading errors so fixed it